test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/../common/pywrap", sys.argv)
+
+import netlib
+import packagelib
+import testlib
+
+no_install = ["rhel4edge", "rhel-8-9", "centos-8-stream"]  # No wireguard-tools
```

Comment:
Please move this into `testVPN`, and make it dynamic. I.e. 

    if m.image in ["rhel4edge", "centos-8-stream"] or m.image.startswith("rhel-8"):

this avoids us having to bump the rhel-8-* value for each new minor release.